cli/get-nexus-cli.sh

```diff
@@ -62,6 +62,8 @@ docker_name="nexus-cli"
 darwin_src_path="/nexus/darwin/nexus"
 linux_src_path="/nexus/linux/nexus"
 
+docker rm -f ${docker_name} &> /dev/null
+
 docker pull ${REPOSITORY}:${VERSION} 1> /dev/null
 docker create --name "$docker_name" ${REPOSITORY}:${VERSION} 1> /dev/null
```

Comment:
@sankar17 we may want to remove the docker image after cli is successfully downloaded ?